iocore/net/SSLSNIConfig.cc

```diff
@@ -92,15 +91,17 @@ SNIConfigParams::loadSNIConfig()
     if (certFile) {
       clientCTX = params->getNewCTX(certFile, keyFile);
     }
-    NextHopProperty *nps        = new NextHopProperty();
-    nps->name                   = ats_strdup(servername);
-    nps->verifyServerPolicy     = item.verify_server_policy;
-    nps->verifyServerProperties = item.verify_server_properties;
-    nps->ctx                    = clientCTX;
-    if (wildcard) {
-      wild_next_hop_table.put(nps->name, nps);
-    } else {
-      next_hop_table.put(nps->name, nps);
+    if (servername) { // a safety check
+      NextHopProperty *nps        = new NextHopProperty();
```

Comment:
This is me harping, but I really, really prefer that you put an empty line after local variable declaration. Not only does it look cleaner, but it really helps clang-format to not do dumb stuff such as aligning the variable declarations with the variable assignments

Reply:
Sure.